src/manager/decisions.ts

```diff
@@ -22,24 +22,45 @@ export function resolve(state: State, storageHandler: WrappedStorageHandler, eve
         cookieDomain)
     }
   }
+
+  let freshDecisions = [] as string[]
```

Comment:
could we do it this way?
```suggestion
  let freshDecisions = [] as  ParsedParam[]
```

then 
```
const extractedFreshDecisions = freshDecisions.concat((state.pageUrl &&
...
```